tests/BusinessLogic/Location/LocationServiceTest.php

```diff
@@ -165,6 +166,37 @@ public function testGetLocationsForNonDropOffService()
         $this->assertEmpty($locations);
     }
 
+    public function testGetLocationsForInvalidPostalCode()
+    {
+        $this->initShippingMethod(true);
+        $this->initWarehouse();
+
+        /** @var LocationService $locationService */
+        $locationService = TestServiceRegister::getService(LocationService::CLASS_NAME);
+        $locations = $locationService->getLocations(1, 'NL', '1011ASZ');
+
+        $this->assertEmpty($locations);
+    }
+
+    public function testGetLocationsForTransformedPostalCode()
+    {
+        $this->httpClient->setMockResponses(array(
+                new HttpResponse(
+                    200, array(), file_get_contents(__DIR__ . '/../Common/ApiResponses/ShippingServices/ShippingServiceDetails-IT-NL.json')
+                ),
+            )
+        );
+
+        $this->initShippingMethod(true);
+        $this->initWarehouse();
+
+        /** @var LocationService $locationService */
+        $locationService = TestServiceRegister::getService(LocationService::CLASS_NAME);
+        $locations = $locationService->getLocations(1, 'NL', '1011AS');
+
+        $this->assertEmpty($locations);
```

Comment:
why empty? should be it formatted to 1011-AS and obtain location?

Reply:
Because there are no drop-off locations for the selected country and postal code selection. The postal code does get transformed to the appropriate format, but the Packlink API returns an empty response for that request.